example_test.go

```diff
@@ -3,11 +3,9 @@ package youtube_test
 import (
 	"flag"
 	"fmt"
+	. "github.com/kkdai/youtube"
```

Comment:
I am kind of curious about this change. because as far as i know, this might violate goimports check. normally goimports will help us sorts imports. maybe you can try to run `make lint` to double check it?

Reply:
my fault,  forget to run golangci-lint before push. fixed!